index.d.ts

```diff
@@ -19,7 +19,7 @@ export type ErrorLike = {
 	code?: string;
 };
 
-export interface Options {
+export type Options = {
```

Comment:
Likely breaking. I can postpone this change via `disable` comment.

The package is still node 14+ so a new major is due